packages/amplify-category-api/src/provider-utils/awscloudformation/utils/check-case-sensitivity.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs-extra';
+import path from 'path';
+
+export const checkCaseSensitivityIssue = async (context: any, category: string, resourceName: string) => {
+  const projectBackendDirPath = context.amplify.pathManager.getBackendDirPath();
+
+  const fsIsCaseSensitive = async () => {
+    const tempFileName = '/tmp/AMPLIFY-TEMP';
```

Comment:
Could this have issues when multiple amplify projects run on the same machine ? 
Does this file need to have a unique path ? e.g. /tmp/${procId} /AMPLIFY-TEMP to avoid side effects ?

Reply:
oh - this file is actually no longer needed because we are going to treat all amplify projects the same regardless of whether the filesystem of their OS could technically support it.